### DISCUSSION TITLE
Dashboard description - is it used anywhere?

### DISCUSSION THREAD

**Discussion by gmwiku** [grafana/grafana#39769] [Dashboards, Panels & Visualizations]
There is field “Description” in “Dashboard settings” → “General”. Is it used anywhere? I would like to see Dashboard descriptions in “Dashboards” → “Manage” and in panel “Dashboard list”. It could be displayed instead of Directory or just as a tip on mouse over.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
